src/AppInstallerCLICore/Public/AppInstallerCLICoreExecute.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include "ExecutionContext.h"
+#include "Command.h"
+
+namespace AppInstaller::CLI
+{
+    int Execute(Execution::Context& context, std::unique_ptr<Command>& command);

```

Comment:
Assuming that the plan is to implement the COM interface classes in another static lib (I support that plan), then many other headers will "need" to be placed in public.  I think it makes sense for Command and Context to be made public, and then COMContext will actually be moved over to that lib.

This function should then be in Command.h, and the "make this command" function should be too.

For now, COMContext can live where you have it in this PR, but at least move this function to Command.h.